schnorr_fun/src/frost.rs

```diff
@@ -620,7 +620,7 @@ impl<H: Digest<OutputSize = U32> + Clone, NG> Frost<H, NG> {
     ///
     /// ## Return value
     ///
-    /// Returns `bool` true if the proof of possession matches the point
+    /// Returns `bool` true if the proof of possession matches the point polynomial
```

Comment:
```suggestion
    /// Returns `bool` true if the proof of possession matches the point
```

The PoP is for the point only.